amlip_docs/rst/demo/wan.rst

```diff
@@ -0,0 +1,189 @@
+.. include:: ../exports/alias.include
+
+.. _demo_inference:
+
+########
+WAN Demo
```

Comment:
I'd change the title, as this looks like it is a WAN example, and it is an edge-inference example in wan

Reply:
Not even using WAN, so change name of the demo and file